manifests/stable/metallb.io_bgppeers.yaml

```diff
@@ -6,18 +6,6 @@ metadata:
   creationTimestamp: null
   name: bgppeers.metallb.io
 spec:
-  conversion:
-    strategy: Webhook
-    webhook:
```

Comment:
Can you move this to a different commit, or even a different pr referencing the bug?

Reply:
* I see it's a separate commit. Let's just reference the bug in the pr then